Ecommerce field calculations (#1119)
rudderlabs/rudderstack-docs: docs/event-spec/ecommerce-events-spec/ordering.mdx

| `products.$.url`        | String   | Contains the URL of the product page                                                         |
| `products.$.image_url`  | String   | Contains the image URL of the product                                                        |

<div class="infoBlock">

> RudderStack SDKs do not calculate the value for any of the above properties like <code class="inline-code">revenue</code>, <code class="inline-code">value</code>, <code class="inline-code">total</code>, <code class="inline-code">Subtotal</code>, etc. These must be provided by the user while triggering the <code class="inline-code">track</code> call. Also, the values associated with these fields are mapped to the destination fields where you want to send the data. Hence, it is helpful to understand how these fields are defined in a specific destination.

Reviewer: ```suggestion
RudderStack's SDKs do not calculate the value for any of the above properties like <code class="inline-code">revenue</code>, <code class="inline-code">value</code>, <code class="inline-code">total</code>, <code class="inline-code">subtotal</code>, etc. You must provide these values while triggering the <code class="inline-code">track</code> call. Also, RudderStack maps the values associated with these fields to the destination fields where you want to send the data. Hence, it is helpful to understand how these fields are defined in a specific destination.
```

Reviewer: - Fixed `subtotal` instead of `Subtotal`.
- Is this callout valid only for the above 4 properties, i.e. `revenue`, `value`, `total`, and `subtotal`, or for some other fields as well? Currently, it feels like there are more.

Author: Its valid for all the properties which can be calculated, like shipping, tax, discount, etc. Thats why there's an etc. at the end
